cmd/packages.go

```diff
@@ -6,6 +6,8 @@ import (
 	"io/ioutil"
 	"os"
 
+	"github.com/anchore/syft/syft/pkg/cataloger"
```

Comment:
smallest nit ever: import loc should be after `"github.com/anchore/syft/syft/format"` ?